src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -148,26 +153,28 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
         className={noTranslate ? 'no-translate' : undefined}
         dir={dir}
         endAdornment={(
-          <InputAdornment position="end">
-            <Tooltip title={showPassword ? getTranslation(translations, 'hide') : getTranslation(translations, 'show')}>
-              <IconButton
-                aria-label={getTranslation(translations, 'visibilityToggleLabel')}
-                aria-pressed={showPassword}
-                aria-controls={name}
-                onClick={handleClickShowPassword}
-                onMouseDown={handleMouseDownPassword}
-                edge="end"
-                sx={{
-                  '&.Mui-focusVisible': {
-                    outlineStyle: 'solid',
-                    outlineWidth: '1px',
-                  },
-                }}
-              >
-                {showPassword ? <EyeOffIcon /> : <EyeIcon />}
-              </IconButton>
-            </Tooltip>
-          </InputAdornment>
+          showPasswordToggleOnSignInPage && (
```

Comment:
I only ask this because I see in Gen2 this is only applied to the identify form, but the other flows that use this component in Gen3 will be impacted (Security question, Password reset, etc). But maybe you all discussed this already and deemed this acceptable since we want to deprecate this eventually?

Reply:
Thanks for the callout and checking to both of you. If the existing gen2 behavior is the same we can keep this the same in gen3.